apollo-router/src/tracer.rs

```diff
@@ -54,7 +54,7 @@ mod test {
     use std::sync::Mutex;
 
     use once_cell::sync::Lazy;
```

Comment:
The changes to this file are required because if the tracer drops out of scope then otel will now avoid doing work by making calls to otel context a no-op.